### PR TITLE
Add FSStore contiguous bug fix note

### PR DESCRIPTION
**Pull request by jakirkham**
Also tidy release notes

xref: https://github.com/zarr-developers/zarr-python/pull/1285#issuecomment-1397468044

cc @joshmoore @madsbk

<hr>

TODO:
* [ ] Add unit tests and/or doctests in docstrings
* [ ] Add docstrings and API docs for any new/modified user-facing classes and functions
* [ ] New/modified features documented in docs/tutorial.rst
* [ ] Changes documented in docs/release.rst
* [ ] GitHub Actions have all passed
* [ ] Test coverage is 100% (Codecov passes)

### CODE REVIEW

**Comment by joshmoore** [comment]
Thanks, @jakirkham! 🙏🏽 but the format looks a bit borked:

Screenshot 2023-01-19 at 20 36 44

**Comment by jakirkham** [comment]
Unfortunately that entry [already has issues]( https://zarr.readthedocs.io/en/latest/release.html#unreleased )

Tried to fix some of them, but can back them out if that is distracting (it is tangential to this PR)

Screen Shot 2023-01-19 at 11 41 46 AM

File: docs/release.rst [hunk 1]
```diff
@@ -14,12 +14,22 @@ Unreleased
     # .. warning::
     #    Pre-release! Use :command:`pip install --pre zarr` to evaluate this release.
 
+
+Major changes
+~~~~~~~~~~~~~
+
 * Improve Zarr V3 support, adding partial store read/write and storage transformers.
   Add two features of the [v3 spec](https://zarr-specs.readthedocs.io/en/latest/core/v3.0.html):
```
Comment (jakirkham) [inline docs/release.rst:22]: Should fix the link

```suggestion
  `v3 spec <https://zarr-specs.readthedocs.io/en/latest/core/v3.0.html>`_.
```

File: docs/release.rst [hunk 2]
```diff
@@ -14,12 +14,22 @@ Unreleased
...
+
+Major changes
+~~~~~~~~~~~~~
+
 * Improve Zarr V3 support, adding partial store read/write and storage transformers.
-  Add two features of the [v3 spec](https://zarr-specs.readthedocs.io/en/latest/core/v3.0.html):
+  `v3 spec <https://zarr-specs.readthedocs.io/en/latest/core/v3.0.html>`_.
   * storage transformers
-  * `get_partial_values` and `set_partial_values`
+  * ``get_partial_values`` and ``set_partial_values``
```
Comment (jakirkham) [inline docs/release.rst:24]: Don't think this list will work, but not sure how to fix it. Naive attempts failed. Maybe we ditch the bullet points and do comma separated?

Comment (joshmoore) [reply docs/release.rst:24]: The list parsing is fixed in @DON-BRAN's theme update so don't knock yourself out there.

**Comment by joshmoore** [comment]
> Unfortunately that entry [already has issues](https://zarr.readthedocs.io/en/latest/release.html#unreleased)

Sorry. I just saw that myself. Didn't mean to offload it on you.

File: docs/release.rst [hunk 3]
```diff
@@ -14,12 +14,25 @@ Unreleased
...
 * Improve Zarr V3 support, adding partial store read/write and storage transformers.
-  Add two features of the [v3 spec](https://zarr-specs.readthedocs.io/en/latest/core/v3.0.html):
-  * storage transformers
-  * `get_partial_values` and `set_partial_values`
+  `v3 spec <https://zarr-specs.readthedocs.io/en/latest/core/v3.0.html>`_.
   By :user:`Jonathan Striebel <jstriebel>`; :issue:`1096`.
 
+  - storage transformers
+
+  - ``get_partial_values`` and ``set_partial_values``
```
Comment (jakirkham) [inline docs/release.rst:27]: So this looks ok in reStructuredText, but Sphinx seems to render it wacky. In any event at least this is consistent with what [we do elsewhere]( https://github.com/zarr-developers/zarr-python/blob/b9e9f5aaa44ca564835c3c242937236ea04aecb2/docs/release.rst#maintenance-4 )

**Comment by jakirkham** [comment]
This is probably as good as we can get

Screen Shot 2023-01-19 at 12 39 14 PM

https://zarr--1325.org.readthedocs.build/en/1325/release.html

**Comment by jakirkham** [comment]
@joshmoore, could you please review when you have a chance?

**Comment by joshmoore** [comment]
Screenshot 2023-01-20 at 12 13 37

Going to call this good. Merging. (@jstriebel can hit us on the head if he wants)

**Pull request merged**